### PR TITLE
Add Riak models for routing blocks.

### PR DESCRIPTION
**Pull request by jerith**
We will need these before implementing the first routing block (or possibly as part of that). See also #277.

### CODE REVIEW

File: go/vumitools/router/models.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        self.status = ROUTER_STOPPED
+
+    def set_status_finished(self):
+        self.archive_status = ROUTER_ARCHIVED
+
+    def __unicode__(self):
+        return self.name
+
+    def get_routing_name(self):
+        return ':'.join((self.conversation_type, self.key))
```
Comment (hodgestar) [inline go/vumitools/router/models.py]: Still says conversation type but I think this whole method can go? At least the equivalent method on conversation doesn't appear to be used anywhere?

**Comment by hodgestar** [comment]
We should have tests for the model methods too, but other than that and the comment about `get_routing_name`, I'm :+1:

File: go/vumitools/router/models.py (hunk 1 above)
Comment (jerith) [inline go/vumitools/router/models.py]: Good point. I'll delete the one on the conversation model too.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**